src/components/modals/Modal.vue

```diff
@@ -67,18 +83,33 @@ export default {
     justify-content: center;
     line-height: 100%;
   }
-  > .close {
+  > .top-right-buttons {
     position: absolute;
-    right: 20px;
+    z-index: 100;
     top: 20px;
-    cursor: pointer;
-    z-index: 5;
-    &:hover {
-      color: red;
+    right: 20px;
+    > .top-right-button {
+      cursor: pointer;
+      width: 28px;
+      &:hover {
+        color: red;
+      }
     }
   }
 }
 
+.maximized {
+  background: rgba(0, 0, 0, 0.95);
+  padding: 0;
+  height: 100%;
+  width: 100%;
+  max-width: 100%;
+  max-height: 100%;
+  display: flex;
+  align-content: center;
+  justify-content: safe center;
```

Comment:
This seems to make me unable to scroll through reference sheets that extend beyond the top of the browser window. For example, load SNV and open the reference sheet in full screen - it will be cut off at the Clockmaker (for me) and I'm unable to scroll further up... :thinking:

Reply:
Hm, I get a scroll bar, but the top is still cut off when the modal is tall. I can also reproduce with the night order by adding loads of players and assigning night-waking characters to them. I'll experiment.

Reply:
seems `justify-content: safe center` is less safe than advertised - according to stack exchange 'safe' is meant to solve this problem

with that line removed the problem goes away, but then modal is vertically aligned to top
![Untitled](https://user-images.githubusercontent.com/14160941/110202390-19d0b300-7e60-11eb-9c38-315ee2e90bef.png)
happy with that?
![Untitled2](https://user-images.githubusercontent.com/14160941/110202388-19381c80-7e60-11eb-9e55-9d533913e4f1.png)

Reply:
Here's an idea: remove the `safe` from `justify-content` as it is only supported on Firefox anyway. Then add a container `div` to Modal.vue which wraps `<slot>` and style that with `max-height: 100%`. This should take care of the cut-off content.

Reply:
It does, but it has a disadvantage:
![Untitled](https://user-images.githubusercontent.com/14160941/110243228-65af5500-7f51-11eb-99ad-3c88ff278f59.png)
Not sure there is a quick way to avoid that - would have to move the top left icons out into the main template?

Reply:
There's an easy fix for that, just add `position: unset` or `position: initial` to the container div.